### DISCUSSION TITLE
Nextjs SSR is not working?

### DISCUSSION THREAD

**Discussion by Lunatix01** [TanStack/query#4094] [Q&A]
### Hello there
I followed the documentation to use react-query in my Nextjs app which one of my pages renders stuff on the server side.
but the thing here I don't know if I am mistaken or what.
when I try to remove this in the getServerSideProps function `await queryClient.fetchQuery(["question"], getQuestion)` it still works.
my Question function takes `locale `and `params.id.` but I'm giving it outside the getServerSideProps function. am I doing something wrong or it actually renders on the server side?

**Comment by Lunatix01** [comment]
[Link of doc](https://tanstack.com/query/v4/docs/guides/ssr)

**Reply by TkDodo** [reply]
idk, show a reproduction?

**Reply by Lunatix01** [reply]
> idk, show a reproduction?

Do you mean the code or the app?

**Reply by TkDodo** [reply]
create a codesandbox reproduction that showcases your questoin
